storage/wdpost_run.go

```diff
@@ -636,14 +636,23 @@ func (s *WindowPoStScheduler) runPoStCycle(ctx context.Context, di dline.Info, t
 				return nil, err
 			}
 
-			postOut, ps, err := s.prover.GenerateWindowPoSt(ctx, abi.ActorID(mid), sinfos, append(abi.PoStRandomness{}, rand...))
+			log.Errorf("actually generating window post")
+			defer func() {
+				if r := recover(); r != nil {
+					log.Errorf("recover: %s", r)
+				}
+			}()
+			postOut, ps, err := s.prover.GenerateWindowPoSt(ctx, abi.ActorID(mid), xsinfos, append(abi.PoStRandomness{}, rand...))
 			elapsed := time.Since(tsStart)
-
-			log.Infow("computing window post", "batch", batchIdx, "elapsed", elapsed)
-
+			log.Infof("computing window post", "batch", batchIdx, "elapsed", elapsed)
```

Comment:
Those should stay as `log.[..]w` - they are used by external tools parsing structured logs